### DISCUSSION TITLE
replace axum with..

### DISCUSSION THREAD

**Discussion by XavDesbordes** [DioxusLabs/dioxus#3202] [Q&A]
Hello,
In case we don't want axum in the stack but another one. 
What would be the best way to deal with it for the web/wasm platform?
Should I use ssr or web or is it possible to replace axum in the fullstack variant?

**Comment by ealmloff** [comment]
You could use the lower level render api in fullstack or contribute to one of the community maintained fullstack adapters for other renderers: https://github.com/DioxusLabs/web-adapters/tree/main/packages/fullstack

**Reply by XavDesbordes** [reply]
Otherwise, I could keep the structure as is, leave the server functions empty so that they are taken into account for the wasm configurations, not launch the axum server and intercept the requests in the new server, given I know the urls and ports, and where the generated wasm executable is. The functions will be implemented under that new server umbrella.

That will enable to keep the -great Dioxus's- dev experience, limit the time to market, while using a better server than axum. 

Maybe I am wrong somewhere in the thinking process, but I will try that.
